### ISSUE TITLE
Fluffy Kitty button does not meow

### ISSUE THREAD

**Issue by kaosjr** [MultiMC/MultiMC5#2692]
System Information
-----------------------------
MultiMC version:
0.6.5
Operating System:
Win 10
Summary of the issue or suggestion:
----------------------------------------------
The Fluffy Kitty button should meow when you press it.

What should happen:
------------------------------
1. You press the kitty button
2. It meows and displays the image

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. Press the Kitty button

2. Notice the lack of meow

3. Be saddened

Suspected cause:
---------------------------
Noone added a meow sound yet.

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)
![image](https://user-images.githubusercontent.com/16449077/59577818-1d178000-907a-11e9-968f-3d4640fe5dcd.png)

Additional Info:
---------------------------
If you need a meow sound, I can meow for you. It's for a good cause!

**Comment by JeSuisUnCaillou** [comment]
Can confirm, I reproduced it on my computer and I was indeed saddened.

I look forward for the meow to be implemented as soon as possible, this is critical.

**Comment by dolphin-cat** [comment]
This is critical we need the Fluffy Kitty button!

**Comment by TheEssem** [comment]
this issue alone makes multimc literally unusable, please fix

**Comment by imichaeldotorg** [comment]
What's the ETA here?

**Comment by Pachin0** [comment]
Just switched to multimc, please fix so I can hear the meow

**Comment by dannyhpy** [comment]
Still no ETA? :(

**Comment by Hexality** [comment]
I would love if there was a meow

Would love even more if both the cat and the sound were customizable sp we can change to our own sprite/sound of something

(I'm not against the cat, I just prefer foxes over cats :3)

**Comment by panda-senpai** [comment]
Confirmed the issue.  Lack of audio feedback leads to a poor experience.  Lack of customizable cat hinders usage.

**Comment by devBob027** [comment]
This problem makes the launcher borderline unusable please fix ASAP

**Comment by CR4KH34D8U2** [comment]
Still broken 3 years later

**Comment by yeeterlol** [comment]
It's been almost 3 years. MultiMC is a horrible launcher, we need the Fluffy Kitty button to MEOW

**Comment by cyphgithub** [comment]
Why is this still an issue? this needs to be resolved asap please

**Comment by goosBanny** [comment]
agreed. it seems that the devs no longer care about community feedback and suggestions. i'm switching back to the default one till this issue is resolved.

**Comment by Hexality** [comment]
Not having this is unacceptable!!???

**Comment by SirFluffDev** [comment]
I've been going through a rough time, thanks to this issue. The only thing keeping me alive is a shred of hope that maybe one day, the fluffy kitty button will meow. Please... let the cat speak his mind.

**Comment by thesaguaro** [comment]
bruh its been (almost) 4 years, fix this rn

**Comment by nin0-dev** [comment]
Its been 4 years we need a meow this makes multimc unusable I will be switching to tlauncher if meow isnt added soon

**Comment by GesundesWasser** [comment]
Can Confirm its still an issue

**Comment by yeeterlol** [comment]
it has been 6 years. im now trans and THERE IS STILL NOT A FLUFFY KITTY BUTTON. what the fuck multimc

**Comment by Orstando** [comment]
I have re-created the PR made by Error504TimeOut.

**Comment by Noobotronic** [comment]
0.7 and it still doesnt meow :(
